package.json

```diff
@@ -4,10 +4,11 @@
   "description": "A React/Redux frontend for Cloudflare plugins",
   "main": "gulpfile.js",
   "scripts": {
-    "test": "mocha src/js/test/*/*.js --compilers js:babel-register",
-    "lint": "gulp lint",
-    "start": "http-server -a localhost -p 8000 -c-1",
-    "postinstall": "bash git-hooks/setup.sh"
```

Comment:
Because Travis CI runs the linter and tests now.  Don't need them running twice.